pydantic/main.py

```diff
@@ -742,7 +742,11 @@ def __getattr__(self, item: str) -> Any:
                 except KeyError as exc:
                     raise AttributeError(f'{type(self).__name__!r} object has no attribute {item!r}') from exc
             else:
-                pydantic_extra = object.__getattribute__(self, '__pydantic_extra__')
+                try:
```

Comment:
Add a comment maybe that this is a guard for using a partially imported module in plugins?

Ok, there is one below. Maybe something short with "see below"?

Reply:
sounds good.